libs/wire-api/test/unit/Test/Wire/API/MLS.hs

```diff
@@ -308,7 +308,7 @@ spawn cp minput = do
        in snd <$> concurrently writeInput readOutput
   case (mout, ex) of
     (Just out, ExitSuccess) -> pure out
-    _ -> assertFailure "Failed spawning process"
+    _ -> assertFailure $ "Failed spawning process\n" <> show mout <> "\n" <> show ex
```

Comment:
Is this related to the current PR or a drive-by debug improvement?

Reply:
the latter (but it's small enough that it didn't bother me too much).